q2_fmt/plugin_setup.py

```diff
@@ -123,7 +128,10 @@
                           ' grouped diversity data and selected comparison.',
     },
     name='Engraftment Pipeline for FMT Analysis',
-    description='',
+    description='Applies group timepoints to Fecal Microbiota Transplant data'
+         'and then applies statistical tests(Wilcoxon rank sum test or '
```

Comment:
Python string concatenation doesn't add spaces, so:
```
("foo" "bar")
```
becomes `"foobar"`.

The way I like to handle this is to start each new string with a space, so that I can see it's a continuation of the last line.